### ISSUE TITLE
Consideration: Expand beyond clustering?

### ISSUE THREAD

**Issue by angelo-hub** [JiriHoffmann/react-native-clusterer#31]
It may be valuable to expand beyond clustering but offloading most geo distance math to a C++ job dedicated to it, could improve main thread performance significantly on react-native applications

**Comment by JiriHoffmann** [comment]
Agreed, that could definitely bring some performance improvements. Unfortunately, I am quite busy at the moment so I won't have time to implement it but I will be more than happy to merge a PR.
